### PR TITLE
[DOC-10355] Document major version selection in Dedicated

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10355] Document major version selection in Dedicated

Extra changes are cleanup after https://github.com/cockroachdb/docs/pull/18754

Previews:
[releases/cloud.md](https://deploy-preview-18759--cockroachdb-docs.netlify.app/docs/releases/cloud.html)
[src/current/cockroachcloud/create-your-cluster.md](https://deploy-preview-18759--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-your-cluster.html)

### CODE REVIEW

File: src/current/_includes/releases/cloud/2024-07-23.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## July 23, 2024
+
+<h3 id="2024-07-23-general-updates"> General updates </h3>
+
+- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version), which defaults to the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version, including subsequent patches after a Pre-Production Preview, are applied automatically.
```
Comment (jaiayu) [inline src/current/_includes/releases/cloud/2024-07-23.md:5]: Should we say additional stable "supported" releases? Or something along those lines, or is it implicit that you can only choose releases that we still support.

Comment (mdlinville) [reply src/current/_includes/releases/cloud/2024-07-23.md:5]: I think it's implicit, you wouldn't be able to select an EOL version even if you have current clusters running that EOL version. And we mention PPP right after, as the other type of release that could be in the list and isn't supported.

File: src/current/cockroachcloud/cluster-overview-page.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ The **Cluster configuration** panel displays the settings you chose during [clus
 |-----------|---------------------------------------------------------------------------------------------------------|
 | Cloud     | The cluster's [cloud provider]({% link cockroachcloud/create-your-cluster.md %}#step-2-select-the-cloud-provider). |
 | Plan type | The [plan type]({% link cockroachcloud/create-your-cluster.md %}#step-1-start-the-cluster-creation-process) used to create the cluster.                                                           |
-| Region(s) | The cluster's [region(s)]({% link cockroachcloud/create-your-cluster.md %}#step-3-configure-region-s-and-node-s).         |
-| Nodes     | The [number of nodes]({% link cockroachcloud/create-your-cluster.md %}#step-3-configure-region-s-and-node-s) the cluster has and how many are live. |
+| Region(s) | The cluster's [region(s)]({% link cockroachcloud/create-your-cluster.md %}#step-3-configure-regions-and-nodes).         |
```
Comment (mdlinville) [inline src/current/cockroachcloud/cluster-overview-page.md:78]:
```suggestion
| Regions   | The cluster's [region(s)]({% link cockroachcloud/create-your-cluster.md %}#step-3-configure-regions-and-nodes).         |
```

File: src/current/cockroachcloud/create-your-cluster.md [hunk 3]
```diff
@@ -154,32 +153,27 @@ The cluster is automatically given a randomly-generated name. If desired, change
...
 
 {{site.data.alerts.callout_danger}}
 Testing releases, including Pre-Production Preview releases, are provided for testing and experimentation only, and are not qualified for production environments and not eligible for support or uptime SLA commitments.
 {{site.data.alerts.end}}
 
-If you install a Pre-Production Preview release, it will be upgraded to each subsequent beta or RC release automatically, before being upgraded to the GA and subsequent patch releases as they become available.
+After the cluster is created, patch releases within its major version are required and are applied automatically. If you install or upgrade to a Pre-Production Preview release, subsequent Pre-Production Preview patch releases, the GA release, and subsequent patches within the major version are applied automatically. To learn more, refer to [Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
```
Comment (mikeCRL) [inline src/current/cockroachcloud/create-your-cluster.md:165]:
```suggestion
After the cluster is created, patch releases within its major version are required and are applied automatically. If you install or upgrade to a Pre-Production Preview release, subsequent Pre-Production Preview patch releases, the GA release, and subsequent patches within the major version are applied automatically. To learn more, refer to the [CockroachDB Cloud Support and Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
```

File: src/current/releases/cloud.md [hunk 4]
```diff
@@ -14,6 +14,12 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## July 23, 2024
+
+<h3 id="2024-07-23-general-updates"> General updates </h3>
+
+- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version), which defaults to the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version, including subsequent patches after a Pre-Production Preview, are applied automatically.
```
Comment (mikeCRL) [inline src/current/releases/cloud.md:21]:
```suggestion
- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version). The default option is the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version are applied automatically.
```

Comment (mikeCRL) [reply src/current/releases/cloud.md:21]: I don't think we need to get into the further detail on PPP. Already documented and linked. Don't want to distract; it's not applicable to most customers.

**Review by mikeCRL — APPROVED** [review]
Approving with minor suggestions

File: src/current/releases/cloud.md [hunk 5]
```diff
@@ -14,6 +14,12 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## July 23, 2024
+
+<h3 id="2024-07-23-general-updates"> General updates </h3>
```
Comment (mdlinville) [inline src/current/releases/cloud.md:19]:
```suggestion
## July 24, 2024

<h3 id="2024-07-24-general-updates"> General updates </h3>
```

File: src/current/cockroachcloud/create-your-cluster.md [hunk 6]
```diff
@@ -154,32 +153,35 @@ The cluster is automatically given a randomly-generated name. If desired, change
 
 ## Step 8. Select the CockroachDB version
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-your-cluster.md:154]: @jaiayu PTAL at this section, which has expanded from when you previously took a look. UI verified using the Education production org, which is not enrolled in the new UI.

**Review by jaiayu — APPROVED** [review]
This looks great!

File: src/current/releases/cloud.md [hunk 7]
```diff
@@ -14,6 +14,12 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## July 24, 2024
```
Comment (mdlinville) [inline src/current/releases/cloud.md:17]:
```suggestion
## July 31, 2024
```

**Pull request merged**